crates/bevy_asset/src/processor/mod.rs

```diff
@@ -76,7 +76,7 @@ use std::{
 use thiserror::Error;
 
 // Needed for doc strings
-#[allow(unused_imports)]
+#[allow(unused_imports, reason = "Needed for documentation links.")]
 use crate::io::{AssetReader, AssetWriter};
```

Comment:
This only affects 8 total documentation links. Looking at other documentation comments, it appears that we prefer to fully qualify the link within the documentation comment rather than doing this.

Reply:
Yep; I'd prefer to fix this but won't block on it being done in this PR.

Reply:
@BenjaminBrienen chuck it in an issue, I'll avoid making that change here :slightly_smiling_face:

Reply:
Done.